Mangosta/ViewControllers/LoginViewController.swift

```diff
@@ -53,6 +55,15 @@ class LoginViewController: UIViewController {
         self.xmppController.connect()
         
     }
+    
+    func showError(errorString: String?) {
+        if let errorString = errorString {
```

Comment:
Hey @kentron , we can avoid this `if let` at all :)
Wouldn't `self.errorLabel.text = errorString` just work?